src/compiler/scala/tools/nsc/backend/jvm/BackendStats.scala

```diff
@@ -9,7 +9,7 @@ package backend.jvm
 import scala.reflect.internal.util.Statistics
 
 // Enable with `-Ystatistics:jvm`
-object BackendStats {
+trait BackendStats {
```

Comment:
Changing this to a trait will make the `timed` method defined in `BackendStats` non-inlineable, so it should be marked `final`. And it should also be moved to a different place, now that it's available in all statistics.

Please check for similar cases around.

Reply:
Good catch. Didn't realise the effect on inlining that this change could bring. I'm checking for similar cases around and I'll move `timed` to `StatisticsInfo`.